### DISCUSSION TITLE
Enable / Disable connection pooling

### DISCUSSION THREAD

**Discussion by janpio** [supabase/supabase#5768] [Questions]
Do I understand the UI under "Database -> Connection Pooling -> Enabled" correctly that I can enable and disable the connection pooling behavior of the database under the `6543` port with this UI element, so when it is enabled I will be talking to a PgBouncer instance, and when it is disabled I will be talking to a direct PostgreSQL instance without connection pooling?

![image](https://user-images.githubusercontent.com/183673/156414632-2d5ac32b-5b63-499f-98d5-a8c57c258fff.png)

(And the `5432` database hostname from from "Settings -> Project Settings, Database -> Connection string" is always unpooled and direct?)

**Comment by GaryAustin1** [comment]
For sure 5432 is direct connection to database and does not ever use pooling.

I believe if you turn off pooling, 6543 is not used at all (versus passthru to database directly), but have not tried it and don't find any documentation on what happens when you turn off pooling. 

But I have seen this in docs:  postgres://postgres:[YOUR-PASSWORD]@db.vdbnhqozmlzdsafsdfsdxwr.supabase.co:6543/postgres?pgbouncer=true
which seems to imply you have an option.

Maybe someone else will come along who knows if it gets passed thru if that is the heart of your question...
